### ISSUE TITLE
Restmode Support

### ISSUE THREAD

**Issue by OSM-Made** [OSM-Made/Orbis-Toolbox#33]
Fix support for entering the suspended state and restarting the toolbox reliably.

**Comment by doclucas** [comment]
Yes, orbis toolbox kills safe mode on my 7.55 and it's the only reason I don't load it, so it's great to see a fix is in progress. :)
Any ETA for a fixed release?

**Comment by doclucas** [comment]
Tested it on fw 9.00 and seem to work, but only reason I don't load it is because rest mode still isn't supported even on this new build and I need to restart the ps4 every time. :(
IMHO, this is the most critical issue with the toolbox.
Any ETA? Do you need a tester?

**Comment by OSM-Made** [comment]
No ETA on a fix as I plan on merging this with another project. I will try to work on restmode support there.

**Comment by Pharaoh2k** [comment]
Cool, thanks!
Do you plan to merge it with the Orbis Suite?

**Comment by OSM-Made** [comment]
Yes! 🙂 That is the plan. I am working on a daemon for the orbis suite api and integrating the toolbox into a pkg file. 😀 There is still lots of work to do but I am slowly picking away at it as I get free time.

**Comment by Pharaoh2k** [comment]
Frankly, the only reason I  personally use the toolbox is to display the CUSA on the game covers :)
Your payload is the only one I know that has that.
I wish there was such a minimal payload that does that but is compatible with rest mode as well...
